src/odemis/gui/xmlh/delmic.py

```diff
@@ -102,7 +102,6 @@
 component.Manager.addXmlHandler(xh_delmic.StreamPanelXmlHandler)
 component.Manager.setMenu(c, 'Delmic', 'Generic Stream Entry', 'StreamPanel', 4)
 
-
```

Comment:
Please don't change that. Now that you're not touching the files in `gui/xmlh/`, they shouldn't have any change at all.